http-server/src/main/java/org/triplea/db/dao/user/UserJdbiDao.java

```diff
@@ -15,9 +15,6 @@
   @SqlQuery("select bcrypt_password from lobby_user where username = :username")
   Optional<String> getPassword(@Bind("username") String username);
 
-  @SqlUpdate("update lobby_user set last_login = now() where username = :username")
-  int updateLastLoginTime(@Bind("username") String username);
```

Comment:
Nothing called this update method, it can be freely removed. It's a bug that we never invoked it. The fix here is a deeper one to get this same value from the access_log table rather than maintaining the last login in two places.